### DISCUSSION TITLE
Delete an issue reaction

### DISCUSSION THREAD

**Discussion by martinf83** [community/community#28525] [Apps, API and Webhooks] [answered]
Hey guys, I have a question with the endpoint of deleting a reaction from an issue, is it possible to delete a reaction from another user in the same repo being me an administrator? I'm trying but it throws me forbidden, when if I try to delete a reaction of mine this gives a 200.
https://docs.github.com/en/rest/reactions#delete-an-issue-reaction

Thanks.

**Comment by martinf83** [comment]
hello someone who can help me?

**Comment by GabyCT** [comment, marked as answer]
It is not possible to delete a reaction from another user even if you are the administrator

**Comment by jfullstackdev** [comment]
if that is your project, try blocking the user

**Comment by martinf83** [comment]
Thanks!!
